src/utils/common.js

```diff
@@ -83,3 +83,8 @@ export const getPrimaryPartnerLogo = (partnerDetails) => {
     alt: partnerDetails.primaryPartner.name,
   };
 };
+
+export const getContactEmail = (contactEmail, adminUsers) => {
```

Comment:
Would it make sense for this to take the whole enterpriseConfig object, in case we add a new email field in the future that we need to factor in?